src/upgradeExecutorPrepareAddExecutorTransactionRequest.ts

```diff
@@ -0,0 +1,59 @@
+import { Address, PublicClient, encodeFunctionData } from 'viem';
+import { upgradeExecutor } from './contracts';
+import {
+  UPGRADE_EXECUTOR_ROLE_EXECUTOR,
+  upgradeExecutorEncodeFunctionData,
+} from './upgradeExecutorEncodeFunctionData';
+
+export type UpgradeExecutorPrepareAddExecutorTransactionRequestParams = {
+  account: Address;
+  upgradeExecutorAddress: Address;
+  executorAccountAddress: Address;
+  publicClient: PublicClient;
+};
+
+export async function upgradeExecutorPrepareAddExecutorTransactionRequest({
+  account,
+  upgradeExecutorAddress,
+  executorAccountAddress,
+  publicClient,
```

Comment:
The same function is used when adding an account to the UpgradeExecutor of the parent chain, and the Orbit chain.